src/ReactiveDomain.UI.Testing/ReactiveDomain.UI.Testing.csproj

```diff
@@ -1,26 +1,25 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="../ci.build.imports" />
   <PropertyGroup>
-    <TargetFrameworks>$(LibTargetFrameworks)</TargetFrameworks>
+    <TargetFrameworks>net472</TargetFrameworks>
```

Comment:
Shouldn't this target `$(TestTargetFrameworks)`?

Reply:
this project only works on that framework, wpf isn't supported in net core/net standard